sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Infrastructure/StorageTestEnvironment.cs

```diff
@@ -15,42 +15,74 @@ namespace Azure.Messaging.EventHubs.Processor.Tests
     public static class StorageTestEnvironment
     {
         /// <summary>The active Azure storage connection for this test run, lazily created.</summary>
-        private static readonly Lazy<StorageScope.StorageProperties> s_activeStorageAccount =
-            new Lazy<StorageScope.StorageProperties>(CreateStorageAccount, LazyThreadSafetyMode.ExecutionAndPublication);
+        private static readonly Lazy<StorageScope.StorageProperties> ActiveStorageAccount =
+            new Lazy<StorageScope.StorageProperties>(EnsureStorageAccount, LazyThreadSafetyMode.ExecutionAndPublication);
+
+        /// <summary>The environment variable value for the storage account connection string, lazily evaluated.</summary>
+        private static readonly Lazy<string> ActiveStorageAccountConnectionString =
+            new Lazy<string>(() => ReadEnvironmentVariable("EVENT_HUBS_BLOBS_CONNECTION_STRING"), LazyThreadSafetyMode.PublicationOnly);
```

Comment:
Was thinking about the names earlier and considering that it may be a good idea to tag these as "overrides" to make sure that its clear you shouldn't use them unless you know what you're doing, because it could cause test failures.   Maybe a pattern like `EVENT_HUBS_OVERRIDE_BLOBS_CONNECTION_STRING` ?      

Does that sound reasonable to you or am I over thinking things?